.github/workflows/deploy-to-gke.yaml

```
@@ -0,0 +1,57 @@
+  REPO_NAME: gearfirst-docker-repo
+  SERVICE_NAME: auth-service
+  DEPLOYMENT_NAME: auth-service-deployment
+
+# 실제 수행할 작업(Job) 정의
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest # 이 작업은 ubuntu 환경에서 실행됩니다.
+    steps:
+      # 1. GitHub 저장소의 코드를 작업 환경으로 가져옵니다.
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)